benefits/core/templates/core/includes/form.html

```diff
@@ -59,7 +59,29 @@
         $("#{{ form.id }}").on("submit", function(e) {
           $(this).trigger("submitting");
         });
-      });
+
+        {% if use_custom_validity %}
```

Comment:
I'm curious if there is another way to derive whether or not this Javascript should be generated here, besides a new context variable? I'm thinking it's one more thing to remember to add when we want this behavior on forms. Is it possible to use the presence of the custom messages or something else?

Reply:
Like maybe a property on the form itself?

Reply:
Does https://github.com/cal-itp/benefits/pull/2045/commits/59c4808ff7fa7dd61909b802eb1d019b6eff1d4f look ok?